deepface/DeepFace.py

```diff
@@ -702,23 +716,23 @@ def represent(
             align=align,
         )
     else:  # skip
-        if isinstance(img_path, str):
-            img = functions.load_image(img_path)
-        elif type(img_path).__module__ == np.__name__:
+        if type(img_path).__module__ == np.__name__:
             img = img_path.copy()
         else:
-            raise ValueError(f"unexpected type for img_path - {type(img_path)}")
+            # Try load. If load error, will raise exception internal
+            img, _ = functions.load_image(img_path)
```

Comment:
Because load_image in fact support many type, and have properly raise. so here I just use `try-except` logic  -- just call it. if failed, it will raise, and it should has almost same effects.